build.gradle

```diff
@@ -1880,9 +1880,10 @@ project(':tools:tools-api') {
 
 project(':tools') {
   archivesBaseName = "kafka-tools"
-
   dependencies {
+    api libs.scalaLibrary
```

Comment:
I think you don't need this one.

Reply:
I believe it's needed for the test cases as the Kafka Harness is written in Scala and `createBrokerProperties` convert from java to scala to setup`TestUtils.createBrokerConfig`

Reply:
Do you get some error removing it? For me it works just fine.

Reply:
You right, my idea was acting up when I remove it but it was wrong state of idea. I remove it along side few other dependencies that we don't need.